delivery_admin/views/react/orders.jsx

```diff
@@ -3,7 +3,7 @@ import $ from 'jquery'
 const date = require('../../../tools/date.js')()
 const globals = require('../../../tools/globals.js')
 
-class Orders extends React.Component {
+export class Orders extends React.Component {
```

Comment:
Check if it already does the export then we can delete "module.exports" at the end of the file